examples/next-js/schema.ts

```diff
@@ -2,10 +2,10 @@
  * Executable schema generated by Grats (https://grats.capt.dev)
  * Do not manually edit. Regenerate by running `npx grats`.
  */
+import { allUsers as queryAllUsersResolver } from "./app/api/graphql/models/User";
```

Comment:
Strange that the ordering shifted here. I fear some non-determinism. Perhaps we should collect and then sort rather than relying on traversal ordering.